src/components/Filter.jsx

```diff
@@ -1,44 +1,57 @@
-const Filter = ({priorityFilter, setPriorityFilter, statusFilter, setStatusFilter, categoryFilter, setCategoryFilter}) => {
+import { FORM_OPTIONS } from "../constants/FormOptions"
+import { STATUS } from "../constants/status"
 
-  return (
-    <div className="filter">
-        <h2>Filter:</h2>
 
-    <div className="filter-options">
-        <div className="filter-priorityOrder">
-            <p>Priority order:</p>
-            <select value={priorityFilter} onChange={(e) => setPriorityFilter(e.target.value)}>
-                <option value="All">All</option>
-                <option value="Urgent">Urgent</option>
-                <option value="High">High</option>
-                <option value="Medium">Medium</option>
-                <option value="Low">Low</option>
-            </select>
-    </div>
+const Filter = ({
+    priorityFilter,
+    setPriorityFilter,
+    statusFilter,
+    setStatusFilter,
+    categoryFilter,
+    setCategoryFilter
+}) => {
+    return (
+        <div className="filter">
+            <div>
+                <p>Priority order:</p>
+                <select 
+                    value={priorityFilter} 
+                    onChange={(e) => setPriorityFilter(e.target.value)}
+                >
+                    <option value={FORM_OPTIONS.PRIORITY.ALL}>{FORM_OPTIONS.PRIORITY.ALL}</option>
+                    <option value={FORM_OPTIONS.PRIORITY.URGENT}>{FORM_OPTIONS.PRIORITY.URGENT}</option>
+                    <option value={FORM_OPTIONS.PRIORITY.HIGH}>{FORM_OPTIONS.PRIORITY.HIGH}</option>
+                    <option value={FORM_OPTIONS.PRIORITY.MEDIUM}>{FORM_OPTIONS.PRIORITY.MEDIUM}</option>
+                    <option value={FORM_OPTIONS.PRIORITY.LOW}>{FORM_OPTIONS.PRIORITY.LOW}</option>
```

Comment:
Tanto aqui quanto nos status e nas categorias, você pode utilizar um map pra criar as options dinamicamente. Seria algo nesse sentido (não validei se está 100% correto, fiz de cabeça).

```suggestion
                    Object.keys(FORM_OPTIONS.PRIORITY).map(key => (
                         <option key={key} value={FORM_OPTIONS.PRIORITY[key]}>
                            {FORM_OPTIONS.PRIORITY[key]}
                          </option>
                    ))
```